src/system/sdl/main.c

```diff
@@ -1062,6 +1062,26 @@ static void handleKeydown(SDL_Keycode keycode, bool down, bool* state, bool* pre
 #endif
 }
 
+char* detect_keyboard_layout() 
+{
+    char q = SDL_GetKeyFromScancode(SDL_SCANCODE_Q);
+    char w = SDL_GetKeyFromScancode(SDL_SCANCODE_W);
+    char y = SDL_GetKeyFromScancode(SDL_SCANCODE_Y);
+
+    char* layout = "unknown";
+
+    if (q == 'q' && w == 'w' && y == 'y') layout = "qwerty"; // US etc.
```

Comment:
can we use enum here instead of strings?
smth like
```c
typedef enum
{
tic_layout_qwerty,
...
} tic_layout;
```

Reply:
Done in 4f54237